### DISCUSSION TITLE
part-ii/2021-4/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#341] [Question comments]
Part II 2021 Algebraic Topology: Paper 1, Section II, 21F 

https://questions.tripos.org/part-ii/2021-4/

**Comment by artie2000** [comment]
<details>
<summary>The construction (draw pictures):</summary>

We want a homotopy inverse $\bar{g}:X/A\to X$ to the natural quotient $\pi:X\to X/A$.

Let $H':A\times I\to X$ be a homotopy from $\text{Id}_A$ to the constant map $c_a:A\to \{a\}$.

Set $f=\text{Id}_X$, and extend $f$ by $H'$ to a homotopy $H:X\times I\to H$ from $\text{Id}_X$ to some $g:X\to X$.

Then $g$ descends by $\sim$ to $\bar{g}:X/A\to X$, which is a section of $\pi:X\to X/A$, and $H$ descends to $\bar{H}$, a homotopy from $\pi\circ\bar{g}$ to $\text{Id}_{X/A}$.
</details>
